training/training_code/train-max-model.sh

```diff
@@ -0,0 +1,207 @@
+#
+# Copyright 2018-2019 IBM Corp. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#!/bin/bash
+
+# uncomment to enable debug output
+#set -x
+
+SUCCESS_RETURN_CODE=0
+TRAINING_FAILED_RETURN_CODE=1
+POST_PROCESSING_FAILED=2
+PACKAGING_FAILED_RETURN_CODE=3
+CUSTOMIZATION_ERROR_RETURN_CODE=4
+ENV_ERROR_RETURN_CODE=5
+
+# --------------------------------------------------------------------
+#  Verify that the required environment variables are defined
+# --------------------------------------------------------------------
+
+# DATA_DIR identifies the directory where the training data is located.
+# The specified directory must exist and be readable.
+if [ -z ${DATA_DIR+x} ]; then
+  echo "Error. Environment variable DATA_DIR is not defined."
+  exit $ENV_ERROR_RETURN_CODE
+fi
+
+if [ ! -d ${DATA_DIR} ]; then 
+  echo "Error. Environment variable DATA_DIR (\"$DATA_DIR\") does not identify an existing directory."
+  exit $ENV_ERROR_RETURN_CODE
+fi
+
+# Verify whether the DATA_DIR contains only one tsv file, and assign it to the TRAINING_DATA variable.
+tsv_count=`ls ${DATA_DIR}/*.tsv 2>/dev/null | wc -l`
+if [ $tsv_count -gt 1 ]; then
+    echo "ERROR: There are multiple .tsv files present in the data directory."
+    exit $ENV_ERROR_RETURN_CODE
+else
+    # Assign it to the TRAIN_DATA variable
+    TRAINING_DATA=`ls ${DATA_DIR}/*.tsv`
+fi
```

Comment:
Are we supporting a validation dataset? Or does the `bert` training code take care of that?

Reply:
Currently it doesn't support a validation dataset, but it could be added in the future.